src/main/java/com/wanikani/api/v2/request/ReviewStatisticsRequest.java

```diff
@@ -1,76 +1,28 @@
 package com.wanikani.api.v2.request;
 
-import com.wanikani.api.v2.model.SubjectType;
+import lombok.Singular;
+import lombok.experimental.SuperBuilder;
 
-import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.List;
 
-import static com.wanikani.api.v2.request.QueryStringUtils.append;
-import static com.wanikani.api.v2.request.QueryStringUtils.appendList;
-import static java.util.stream.Collectors.toList;
-
-public class ReviewStatisticsRequest implements Request {
-
-    private final String queryString;
-
-    public ReviewStatisticsRequest(String queryString) {
-        this.queryString = queryString;
-    }
-
-    public static Builder builder() {
-        return new Builder();
-    }
-
+@SuperBuilder(toBuilder = true)
+public class ReviewStatisticsRequest extends CollectionRequest {
+    @Singular
+    private List<Long> subjectIds;
+    @Singular
+    private List<String> subjectTypes;
+    private Integer percentagesGreaterThan;
+    private Integer percentagesLessThan;
+    private Boolean hidden;
```

Comment:
I believe these could be `final` (same for the fields in `StudyMaterialsRequest`).